COLLABORATOR_GUIDE.md

```diff
@@ -87,6 +87,25 @@ All pull requests that modify executable code should be subjected to
 continuous integration tests on the
 [project CI server](https://ci.nodejs.org/).
 
+Here are some useful CI jobs:
```

Comment:
@morrme if it seems reasonable to you then go for it. The only downside to implementing suggested changes is that if lots of people disagree you might have to change it back. This one doesn't seem very controversial.